web-apps/genesys-spark-examples/src/component-viewer/app.js

```diff
@@ -140,6 +144,13 @@ export async function bootstrap(exampleCode, callback) {
     setNewTheme('gux-dark-theme', preview, darkThemeButton, buttons)
   );
 
+  editorToggle.addEventListener('click', () =>
+    editor.classList.toggle('editor-hidden')
+  );
```

Comment:
Just noticed a bug because this is listening for `click`. You can click on the toggle label the that will not toggle the `gux-toggle` state but it will toggle showing or hiding the editor state.

you can list to the components custom `check` event (https://github.com/MyPureCloud/genesys-spark/tree/main/packages/genesys-spark-components/src/components/stable/gux-toggle#events)

It will be something like 
```
  editorToggle.addEventListener('check', (active) =>
    if (active) {
      editor.classList.remove('editor-hidden')
    } else {
      editor.classList.add('editor-hidden')
    }
  );
```

Reply:
Updated with the check event. I kept the toggle because it defaults to the editor being visible so it still works that way. If it ends up being useful for people, we could have a cookie to hide the editor between pages either.

Reply:
I think it would be better to actually link the the toggle event details to the class changes instead of keeping the `classList.toggle` and just hoping they always stay insync.

Reply:
I was worried you might bring that up... :sweat_smile: Updated now